packages/core/src/sync-store/sqlite/store.ts

```diff
@@ -2059,10 +2072,10 @@ export class SqliteSyncStore implements SyncStore {
                 "transactionReceipts.type as txr_type",
               ]),
           )
-          .where("events.checkpoint", ">", cursor)
-          .where("events.checkpoint", "<=", encodedToCheckpoint)
+          .where("events.checkpoint", ">", fromCursor)
+          .where("events.checkpoint", "<=", toCursor)
```

Comment:
Worth double-checking the exclusivity rules here again (I think we're good though).